etcdserver/server.go

```diff
@@ -156,7 +157,8 @@ type EtcdServer struct {
 	id         types.ID
 	attributes Attributes
 
-	cluster *cluster
+	cluster   *cluster
+	snapStore *snapshotStore
```

Comment:
I could move it into etcdserver.raftNode because it always interacts with rafthttp and raftStorage.